tests/testTriggers.go

```diff
@@ -654,6 +656,79 @@ func TriggerRebootNodes(contexts *[]*scheduler.Context, recordChan *chan *EventR
 	})
 }
 
+// TriggerCrashNodes crashes node on which apps are running
+func TriggerCrashNodes(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: CrashNode,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	Step("get all nodes and crash one by one", func() {
+		nodesToCrash := node.GetWorkerNodes()
+
+		// Crash node and check driver status
+		Step(fmt.Sprintf("crash node one at a time from the node(s): %v", nodesToCrash), func() {
+			// TODO: Below is the same code from existing nodeCrash test
+			for _, n := range nodesToCrash {
+				if n.IsStorageDriverInstalled {
+					Step(fmt.Sprintf("crash node: %s", n.Name), func() {
+						taskStep := fmt.Sprintf("crash node: %s.", n.MgmtIp)
+						event.Event.Type += "<br>" + taskStep
+						err := Inst().N.CrashNode(n, node.CrashNodeOpts{
+							Force: true,
+							ConnectionOpts: node.ConnectionOpts{
```

Comment:
please make necessary changes in longevity_test.go to trigger this test.
Also attach the successful run result

Reply:
made necessary changes to longevity_test.go; Please find the attached run lo
[longevity.log](https://github.com/portworx/torpedo/files/7888457/longevity.log)
g